include/mbedtls/ctr_drbg.h

```diff
@@ -364,8 +377,9 @@ void mbedtls_ctr_drbg_set_entropy_len( mbedtls_ctr_drbg_context *ctx,
  * \brief               This function sets the amount of entropy grabbed
  *                      as a nonce for the initial seeding.
  *
- * Call this function before calling mbedtls_ctr_drbg_seed() to read
- * a nonce from the entropy source during the initial seeding.
+ *                      Call this function before calling
```

Comment:
I've squashed your two commits and it seems those changes have not be reverted. As we are at it, could you squash your two commits? That way it will be clearer what is eventually changed by the PR.